README.md

```diff
@@ -20,23 +20,20 @@ PyO3 supports Python 3.6 and up. The minimum required Rust version is 1.41.
 
 PyPy is also supported. Some minor features are unavailable on PyPy - please refer to the [pypy section in the guide](https://pyo3.rs/latest/building_and_distribution/pypy.html) for more information.
 
-You can either write a native Python module in Rust, or use Python from a Rust binary.
+You can use PyO3 to write a native Python module in Rust, or to embed Python in a Rust binary. The following sections explain each of these in turn.
 
-However, on some OSs, you need some additional packages. E.g. if you are on _Ubuntu 18.04_, please run
+### Using Rust from Python
 
-```bash
-sudo apt install python3-dev python-dev
-```
+PyO3 can be used to generate a native Python module. The easiest way to try this out for the first time is to use [`maturin`](https://github.com/PyO3/maturin). `maturin` is a tool for building and publishing Rust-based Python packages with minimal configuration. The following steps set up some files for an example Python module, install `maturin`, and then show how build and import the Python module.
 
-## Using Rust from Python
-
-PyO3 can be used to generate a native Python module.
+First, create a new folder (let's call it `string_sum`) containing the following two files:
 
 **`Cargo.toml`**
 
 ```toml
 [package]
 name = "string-sum"
+authors = "PyO3 Contributors"
```

Comment:
Is `authors` not usually a list? And now it's optional anyway...

Reply:
I think we should just remove it. https://github.com/rust-lang/rfcs/blob/master/text/3052-optional-authors-field.md

Reply:
Yep, this is a typo from me on top of the fact I was erroneously testing with an old `maturin` version which still required `authors`. Will remove 👍